### PR TITLE
Update RubyMC.net server mappings

### PR DESCRIPTION
**Pull request by Plunzi**
## Type of change

- [ ] Add new directory for a server
- [x] Update directory of a server
- [ ] Remove directory of a server
- [ ] Documentation Update

## Checklist

- [x] I have read the [README](https://github.com/LabyMod/server-media/blob/master/README.md) doc
- [x] I have compressed the images appropriately (e.g. on https://tinypng.com)
- [x] I have created the manifest.json with the required values

### CODE REVIEW

File: minecraft_servers/rubymc/manifest.json [hunk 1]
```diff
@@ -7,46 +7,42 @@
       "en"
    ],
    "social":{
-      "web_shop":"https://shop.rubymc.de",
-      "discord":"https://discord.gg/Wu4PSKaQA2",
-      "youtube":"https://www.youtube.com/channel/UCdP40AdmG_ZMM6A19dJN8fw"
+      "web_shop":"https://shop.rubymc.net",
```
Comment (DoJapHD) [inline minecraft_servers/rubymc/manifest.json:10]: Not reachable

File: minecraft_servers/rubymc/manifest.json [hunk 2]
```diff
@@ -7,46 +7,42 @@
...
+      "discord":"https://discord.gg/rubymc",
+      "youtube":"https://www.youtube.com/@rubymcnet"
```
Comment (DoJapHD) [inline minecraft_servers/rubymc/manifest.json:12]: Not found

**Review by DoJapHD — CHANGES_REQUESTED** [review]

**Comment by Plunzi** [comment]
@DoJapHD The Web-Shop is currently undergoing some changes therefore it is not reachable, therefore the values entered are correct. Thanks for the review anyways.

**Comment by DoJapHD** [comment]
How long will that take?

**Comment by Plunzi** [comment]
Unsure, I'd guess it will take some more weeks.

**Comment by DoJapHD** [comment]
Then remove from PR, you can add again it if it's done

**Comment by DoJapHD** [comment]
Inactive PR

**Pull request closed without merging**